### PR TITLE
Feature/extracting support module

### PR DESCRIPTION
**Pull request by mrsasha**
moves the files from support package inside `androidbound` module to `androidbound-support` module. AB module now has no support dependencies except RV support library (small enough).

### CODE REVIEW

File: androidbound-support/src/main/java/solutions/alterego/androidbound/support/android/ui/BindableCompatEditText.java [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+    public Typeface getTypeface() {
+        return super.getTypeface();
+    }
+
+    public void setTypeface(Typeface font) {
+        super.setTypeface(font);
+    }
+
+    public ColorStateList getTextColor() {
```
Comment (bblackbelt) [inline androidbound-support/src/main/java/solutions/alterego/androidbound/support/android/ui/BindableCompatEditText.java]: shouldn't this return an int ? to match its setter

File: androidbound-support/src/main/java/solutions/alterego/androidbound/support/android/ui/BindableCompatEditText.java [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+    public int getBackgroundDrawable() {
+        return 0;
+    }
+
+    public void setBackgroundDrawable(Drawable res) {
+        super.setBackgroundDrawable(res);
+    }
+
+    public Typeface getTypeface() {
```
Comment (bblackbelt) [inline androidbound-support/src/main/java/solutions/alterego/androidbound/support/android/ui/BindableCompatEditText.java]: get and set typeface are just noise. remove them please

File: androidbound-support/src/main/java/solutions/alterego/androidbound/support/android/ui/BindableCompatTextView.java [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+    public int getBackgroundDrawable() {
+        return 0;
+    }
+
+    public void setBackgroundDrawable(Drawable res) {
+        super.setBackgroundDrawable(res);
+    }
+
+    public Typeface getTypeface() {
```
Comment (bblackbelt) [inline androidbound-support/src/main/java/solutions/alterego/androidbound/support/android/ui/BindableCompatTextView.java]: see above

**Review by bblackbelt — CHANGES_REQUESTED** [review]

File: androidbound-support/src/main/java/solutions/alterego/androidbound/support/android/ui/BindableCompatEditText.java (hunk 2 above)
Comment (mrsasha) [reply androidbound-support/src/main/java/solutions/alterego/androidbound/support/android/ui/BindableCompatEditText.java]: you're right, removed them.

File: androidbound-support/src/main/java/solutions/alterego/androidbound/support/android/ui/BindableCompatEditText.java (hunk 1 above)
Comment (mrsasha) [reply androidbound-support/src/main/java/solutions/alterego/androidbound/support/android/ui/BindableCompatEditText.java]: good catch, thx!

**Review by bblackbelt — APPROVED** [review]

**Pull request merged**